README.md

```diff
@@ -51,6 +51,39 @@ documentation.
 3. Create a new tag
 4. Publish a wheel `python setup.py bdist_wheel upload`
 
+## Option 2 - Developing/Testing changes to this theme
+
+If you prefer to stay within github universe and provide changes via pull requests, you can do
+```bash
+#Go to target directory, where you want clone cakephp/docs and cakephp/sphinxtheme into
+#In our case we do it in the users home directory 
+cd ~
+
+#Clone the docs (follow installation guidelines of cakephp/docs)
+git clone git@github.com:cakephp/docs
+cd cake_docs
+
+#Note/recommendation: better work with your own fork of cakephp/docs  
+
+#Install sphinx and other requirements for cakephp/docs
+pip install -r requirments.txt
```

Comment:
I think you need an ending triple backtick here.

Reply:
updated patch-2 has now 2 code blocks - ending triple backticks and new bash block added (also fixed type in requirements.txt)